recipes-app/iot2050-event-record/WATCHDOG.md

```diff
@@ -0,0 +1,127 @@
+Due to the fact that the watchdog is non-stoppable, getting a watchdog event always requires opening the watchdog and feeding the watchdog. So `event-record` doesn't contain the watchdog initially.
```

Comment:
Line too long

Reply:
, the watchdog event could not be included into the `iot2050-event-record` service.

Reply:
ACK and done.